Assets/HoloToolkit/Utilities/Scripts/CameraCache.cs

```diff
@@ -0,0 +1,30 @@
+﻿using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    public static class CameraCache
+    {
+        private static Camera cachedCamera;
+
+        /// <summary>
+        /// Returns a cached reference to the main camera and uses Camera.main if it hasn't been cached yet.
+        /// </summary>
+        public static Camera main
```

Comment:
I had the same in mind to keep it the same as the unity call. But that's a good point to see a larger difference. One could mistake `CameraCache.main` for `Camera.main` when skimming over the code.